specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of supported prebuilt entities.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringPrebuiltEntities"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Supported Prebuilt Entities": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetSupportedPrebuiltEntities.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/training-config-versions": {
+      "get": {
+        "description": "Lists the support training config version for a given project type.",
+        "operationId": "ConversationalAnalysisAuthoring_ListTrainingConfigVersions",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ConversationalAnalysisAuthoringProjectKindInQuery"
+          },
+          {
+            "$ref": "common.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/MaxPageSizeParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of config versions.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringTrainingConfigVersions"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Supported Training Config Versions": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetSupportedTrainingConfigVersions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ConversationalAnalysisAuthoringCompositionSetting": {
+      "enum": [
+        "returnLongestOverlap",
+        "requireExactOverlap",
+        "separateComponents",
+        "combineComponents"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CompositionSetting",
+        "modelAsString": true
+      },
```

Comment:
The enum values should be described. See https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-enum